lighthouse-core/audits/dobetterweb/inspector-issues.js

```diff
@@ -175,8 +175,16 @@ class IssuesPanelEntries extends Audit {
     }
     const cspIssues = issues.contentSecurityPolicy.filter(issue => {
       // kTrustedTypesSinkViolation and kTrustedTypesPolicyViolation aren't currently supported by the Issues panel
-      return issue.contentSecurityPolicyViolationType !== 'kTrustedTypesSinkViolation' &&
-        issue.contentSecurityPolicyViolationType !== 'kTrustedTypesPolicyViolation';
+      return (
+        issue.contentSecurityPolicyViolationType !== 'kTrustedTypesSinkViolation' &&
+        issue.contentSecurityPolicyViolationType !== 'kTrustedTypesPolicyViolation' &&
+        // filter csp issue from tap-targets gatherer
+        !(
+          issue.blockedURL === undefined &&
```

Comment:
I didn't confirm that `blockedURL` would only be blank from code that Lighthouse injects. Does that seem to be the case?

Reply:
I got that from: [#11862#issuecomment-788351373](https://github.com/GoogleChrome/lighthouse/issues/11862#issuecomment-788351373)